deploy/local/docker-compose-dbsetup.yml

```diff
@@ -3,9 +3,10 @@ version: '3.4'
 services:
   dbsetup:
     image: ${AMBUDA_IMAGE}
-    command: "/app/scripts/start-database.sh"
+    command: "/app/scripts/setup-data.sh"
```

Comment:
nit: `setup` as a single word is a noun, as opposed to `set up`. Maybe `initialize-data`?

Reply:
done